packages/spaces/src/Spaces.js

```diff
@@ -196,34 +200,98 @@ Spaces.propTypes = {
 
 Spaces.defaultProps = {
   query: `
-    query($ids: [String!], $payerIDs: [String!], $types: [String!], $page: Int){
-      spaces(ids: $ids, payerIDs: $payerIDs, types: $types, page: $page){
-        totalCount
-        perPage
-        page
-        spaces{
-          id
-          name
-          description
-          link {
+    query configurationFindMany($ids: [String!], $payerIDs: [ID!], $types: [TypeEnum!]){
+      configurationPagination(filter: { ids: $ids, payerIds: $payerIDs, types: $types }){
+        pageInfo{
+          pageCount
+          currentPage
+          perPage
+          itemCount
+        }
+        items {
+          ...on Configuration{
+              name
+              description
+              payerIDs
+              parentIDs
+              metadataPairs{
+                name
+                value
+              }
+          }
+          
+          ... on Node {
+            id
+          }
+          
+          ... on Alert {
+            link{
+              url
+            }
+
+          }
+          
+          ... on Container {
+            link{
+              url
+            }
+            images{
+              tile
+              promotional
+              logo
+              billboard
+            }
+          }
+          
+          ... on PayerSpace {
+            link{
+              url
+            }
+            images{
+              tile
+              logo
+              billboard
+            }
             url
           }
-          payerIDs
-          parentIDs
-          metadata{
-            name
-            value
+          
+          ... on Application {
+            link{
+              url
+            }
+          }
+          
+          ... on Resource {
+            link{
+              url
+            }
+          }
+          
+          ... on Navigation {
+            images{
+              promotional
+            }
+          }
+          
+          ... on Learning {
+            images {
+              promotional
+            }
           }
-          images{
-            name
-            value
+          
+          ... on Proxy {
+            url
+          }
+          
+          ... on File {
+            url
           }
-          url
+
         }
       }
     }
   `,
-  variables: { types: ['space'] },
+  variables: { types: ['PAYERSPACE'] },
```

Comment:
Do we need to add the other configuration types as variables here?

Reply:
not for the default. devs who implement this package can override that prop.

Reply:
Gotcha, thanks for clarifying